### PR TITLE
Add submenu chevron

### PR DESCRIPTION
**Pull request by lenahong**
### Platforms Impacted
- [ ] iOS
- [ ] macOS
- [X] win32 (Office)
- [ ] windows
- [ ] android

### Description of changes
- Added the Office commoncontrolchevronforward.12.svg to our bundle
- Add a chevron Icon slot to SubmenuItem
- Add a right aliged stack to SubmenuItem for chrvron

### Verification

nestes menu chevron

### Pull request checklist

This PR has considered (when applicable):
- [ ] Automated Tests
- [X] Documentation and examples
- [ ] Keyboard Accessibility
- [ ] Voiceover
- [ ] Internationalization and Right-to-left Layouts

### CODE REVIEW

File: packages/components/ContextualMenu/src/SubmenuItem.tsx [hunk 1]
```diff
@@ -75,6 +76,18 @@ export const SubmenuItem = compose<SubmenuItemType>({
...
+    const svgProps: SvgIconProps = {
+      src: ChevronSvg,
+      viewBox: '0 0 2048 2048',
+    };
+
+    const chevronProps: IconProps = {
+      svgSource: svgProps,
+      width: 12,
+      height: 12,
+      color: 'red'
```
Comment (rurikoaraki) [inline packages/components/ContextualMenu/src/SubmenuItem.tsx:88]: Is the "red" intentional for the svg?

**Review by rurikoaraki — APPROVED** [review]
Looks good to me! Just had one question, otherwise looks good.

File: packages/components/ContextualMenu/src/SubmenuItem.settings.ts [hunk 2]
```diff
@@ -15,26 +15,35 @@ export const settings: IComposeSettings<SubmenuItemType> = [
...
-        minWidth: 80,
-        justifyContent: 'center',
-      },
+        justifyContent: 'flex-start',
+      }
+    },
+    rightstack: {
+      style: {
+        display: 'flex',
+        paddingEnd: 5,
```
Comment (samuelfreiberg) [inline packages/components/ContextualMenu/src/SubmenuItem.settings.ts:40]: Does the UI look consistent across all DPIs?

File: packages/components/ContextualMenu/src/SubmenuItem.tsx (hunk 1 above)
Comment (lenahong) [reply packages/components/ContextualMenu/src/SubmenuItem.tsx:88]: oops! I'll remove it

**Comment by samuelfreiberg** [comment]
@lenahong  I'm not sure this falls on your plate or design, but I noticed the color for the highlighted context menu entry in the image above (The "Nested Menu" one) doesn't contrast that much with the surrounding entries. I used Colour Contrast Analyser to see if it's compliant with a11y guidelines, and it doesn't seem to be.

Adding @ksiler for input as well

**Review by samuelfreiberg — CHANGES_REQUESTED** [review]
@lenahong  I'm not sure this falls on your plate or design, but I noticed the color for the highlighted context menu entry in the image above (The "Nested Menu" one) doesn't contrast that much with the surrounding entries. I used Colour Contrast Analyser to see if it's compliant with a11y guidelines, and it doesn't seem to be.

Adding @ksiler for input as well

**Comment by lenahong** [comment]
> @lenahong I'm not sure this falls on your plate or design, but I noticed the color for the highlighted context menu entry in the image above (The "Nested Menu" one) doesn't contrast that much with the surrounding entries. I used Colour Contrast Analyser to see if it's compliant with a11y guidelines, and it doesn't seem to be.
> 
> Adding @ksiler for input as well

hmm interesting! I current have the hovered background set as "menuItemBackgroundHovered"

File: packages/components/ContextualMenu/src/SubmenuItem.settings.ts (hunk 2 above)
Comment (lenahong) [reply packages/components/ContextualMenu/src/SubmenuItem.settings.ts:40]: I believe so!

**Comment by ksiler** [comment]
> > @lenahong I'm not sure this falls on your plate or design, but I noticed the color for the highlighted context menu entry in the image above (The "Nested Menu" one) doesn't contrast that much with the surrounding entries. I used Colour Contrast Analyser to see if it's compliant with a11y guidelines, and it doesn't seem to be.
> > Adding @ksiler for input as well
> 
> hmm interesting! I current have the hovered background set as "menuItemBackgroundHovered"

I agree that the color contrast doesn't meet WCAG 1.4.3 requirements but this problem is present in other controls throughout our tester (check out the background of our buttons vs the background on hover example). I wouldn't hold up this PR just on this but we should probably create an issue to address it. I'll go ahead and create one.

**Review by samuelfreiberg — APPROVED** [review]
We should make a task to address the contrasting issues in our repro :)

**Comment by ksiler** [comment]
> > > @lenahong I'm not sure this falls on your plate or design, but I noticed the color for the highlighted context menu entry in the image above (The "Nested Menu" one) doesn't contrast that much with the surrounding entries. I used Colour Contrast Analyser to see if it's compliant with a11y guidelines, and it doesn't seem to be.
> > > Adding @ksiler for input as well
> > 
> > 
> > hmm interesting! I current have the hovered background set as "menuItemBackgroundHovered"
> 
> I agree that the color contrast doesn't meet WCAG 1.4.3 requirements but this problem is present in other controls throughout our tester (check out the background of our buttons vs the background on hover for example). I wouldn't hold up this PR just on this but we should probably create an issue to address it. I'll go ahead and create one.

I want to make one correction to what I said above. The [WCAG 1.4.3](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html) requirement is for text only (in this case, it'd be referring to the text "Nested Menu" and how it contrasts with the background color on hover). It does not apply to the contrast between the background of the hovered state of the entry and the background of the entry above it. [WCAG 1.4.11](https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast.html) would cover this instead.

So it does meet WCAG 1.4.3 but violates WCAG 1.4.11. I'll still create an issue for it but there my be some discussion as to whether or not we'll fix it. I'm not sure what our bar is here.

**Comment by ksiler** [comment]
Issue created here for color contrast: #745

**Pull request merged**